vault/logical_system.go

```diff
@@ -4606,9 +4606,10 @@ func (b *SystemBackend) pathInternalOpenAPI(ctx context.Context, req *logical.Re
 						Description: "Path that the backend was mounted at",
 						In:          "path",
 						Schema: &framework.OASSchema{
-							Type: "string",
+							Type:    "string",
+							Default: strings.TrimRight(mount, "/"),
 						},
-						Required: true,
+						Required: false,
```

Comment:
@averche This change re-adds something we previously reverted... it violates the OpenAPI spec.

When "in" is "path", "required" MUST be true.

Reply:
You are right, I'll put in a PR to fix this. Thanks!

Reply:
#18955